conf.d/tmux.fish

```diff
@@ -58,7 +58,7 @@ alias tmux=_fish_tmux_plugin_run
 
 set -q fish_tmux_autostarted || set fish_tmux_autostarted false
 if status is-interactive && ! fish_is_root_user
-    if test -z $TMUX && test $fish_tmux_autostart = true && test -z $INSIDE_EMACS && test -z $EMACS && test -z $VIM && test ! $TERM_PROGRAM = 'vscode'
+    if test -z $TMUX && test $fish_tmux_autostart = true && test -z $INSIDE_EMACS && test -z $EMACS && test -z $VIM && test ! "$TERM_PROGRAM" = 'vscode'
```

Comment:
can u change it to `test "$TERM_PROGRAM != "vscode"`?

Reply:
I think it's OK.
Sorry, I never PR before, maybe I did something wrong.

Reply:
Nah there's nothing wrong here, it's easier to understand with "test this is not equal to that" than "test if not this equal to that" :D